packages/core/lib/src/styles/app_themes.dart

```diff
@@ -77,3 +77,11 @@ class AppThemes {
     );
   }
 }
+
+/// Theme extension with helper getters and methods
+extension ThemeExtension on ThemeData {
```

Comment:
@jenshor What do you think about this?
I figured it's easier to use 
```
Theme.of(context).isDark
``` 

instead of 
```
Theme.of(context).brightness == Brightness.dart
````

Reply:
That's ok for me